app/controllers/v3/roles_controller.rb

```diff
@@ -203,4 +214,8 @@ def lookup_user_guid_in_uaa(username, given_origin, creating_space_role: false)
   def uaa_username_lookup_client
     CloudController::DependencyLocator.instance.uaa_username_lookup_client
   end
+
+  def org_managers_can_create_users?
+    VCAP::CloudController::Config.config.get(:allow_user_creation_by_org_manager) && FeatureFlag.raise_unless_enabled!(:set_roles_by_username)
```

Comment:
This implementation is different from `org_managers_can_create_users?` in `v3/users_controller.rb`, is that intended?

Reply:
Yes that's intended. For the `/v3/users` endpoint we have to check whether the user is OrgManager in any org. For `/v3/roles` those checks are done elsewhere as we have the information about the org, in which the user tries to create a role in.